<ui version="4.0" >
 <class>OTRPrefsUI</class>
 <widget class="QWidget" name="OTRPrefsUI" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>548</width>
    <height>495</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>OTR Plugin Preferences</string>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <widget class="QTabWidget" name="twSettings" >
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="settings" >
      <attribute name="title" >
       <string>&amp;Settings</string>
      </attribute>
      <layout class="QGridLayout" >
       <item row="0" column="0" >
        <widget class="QGroupBox" name="groupBox1" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title" >
          <string>Private Keys</string>
         </property>
         <layout class="QGridLayout" >
          <item row="0" column="0" >
           <widget class="QLabel" name="textLabel2" >
            <property name="text" >
             <string>Account:</string>
            </property>
            <property name="wordWrap" >
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="1" >
           <widget class="QComboBox" name="cbKeys" >
            <property name="maximumSize" >
             <size>
              <width>400</width>
              <height>32767</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="0" column="2" >
           <widget class="QPushButton" name="btGenFingerprint" >
            <property name="text" >
             <string>&amp;Generate</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" >
           <widget class="QLabel" name="label" >
            <property name="text" >
             <string>Fingerprint:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1" >
           <widget class="QLabel" name="tlFingerprint" >
            <property name="frameShape" >
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="text" >
             <string>No Fingerprint</string>
            </property>
            <property name="alignment" >
             <set>Qt::AlignVCenter</set>
            </property>
            <property name="wordWrap" >
             <bool>false</bool>
            </property>
            <property name="textInteractionFlags" >
             <set>Qt::LinksAccessibleByMouse|Qt::NoTextInteraction|Qt::TextSelectableByMouse</set>
            </property>
           </widget>
          </item>
          <item row="1" column="2" >
           <spacer>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" >
             <size>
              <width>71</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0" >
        <widget class="QGroupBox" name="gbPolicy" >
         <property name="title" >
          <string>Default OTR Policy</string>
         </property>
         <layout class="QGridLayout" >
          <item row="0" column="0" >
           <widget class="QRadioButton" name="kcfg_rbAlways" >
            <property name="toolTip" >
             <string>Always encrypt messages</string>
            </property>
            <property name="text" >
             <string>Al&amp;ways</string>
            </property>
            <property name="checked" >
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="1" >
           <widget class="QRadioButton" name="kcfg_rbManual" >
            <property name="toolTip" >
             <string>Encrypt messages if the other side requests an OTR connection</string>
            </property>
            <property name="text" >
             <string>&amp;Manual</string>
            </property>
            <property name="checked" >
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0" >
           <widget class="QRadioButton" name="kcfg_rbOpportunistic" >
            <property name="toolTip" >
             <string>Encrypt messages automatically if the other side supports OTR</string>
            </property>
            <property name="text" >
             <string>&amp;Opportunistic</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1" >
           <widget class="QRadioButton" name="kcfg_rbNever" >
            <property name="toolTip" >
             <string>Never encrypt messages</string>
            </property>
            <property name="text" >
             <string>Ne&amp;ver</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0" >
           <widget class="QCheckBox" name="kcfg_cbAllowUnenc" >
            <property name="toolTip" >
             <string>Show unencrpyted Messages</string>
            </property>
            <property name="text" >
             <string>Show unencrpyted Messages</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1" >
           <widget class="QCheckBox" name="kcfg_cbEndOnLeave" >
            <property name="toolTip" >
             <string>End your OTR conversation automatically instead of manually having to stop otr</string>
            </property>
            <property name="text" >
             <string>End OTR connection on conversation leave</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="0" >
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>161</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="fingerprints" >
      <attribute name="title" >
       <string>K&amp;nown Fingerprints</string>
      </attribute>
      <layout class="QGridLayout" >
       <item row="0" column="0" colspan="2" >
        <widget class="QTableWidget" name="tbFingerprints" >
         <property name="focusPolicy" >
          <enum>Qt::WheelFocus</enum>
         </property>
         <property name="editTriggers" >
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="selectionMode" >
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior" >
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <property name="columnCount" >
          <number>5</number>
         </property>
         <column>
          <property name="text" >
           <string>Contact ID</string>
          </property>
         </column>
         <column>
          <property name="text" >
           <string>Status</string>
          </property>
         </column>
         <column>
          <property name="text" >
           <string>Verified</string>
          </property>
         </column>
         <column>
          <property name="text" >
           <string>Fingerprint</string>
          </property>
         </column>
         <column>
          <property name="text" >
           <string>Protocol</string>
          </property>
         </column>
        </widget>
       </item>
       <item row="1" column="0" >
        <widget class="QPushButton" name="btVerify" >
         <property name="enabled" >
          <bool>true</bool>
         </property>
         <property name="text" >
          <string>&amp;Verify Fingerprint</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" >
        <widget class="QPushButton" name="btForget" >
         <property name="enabled" >
          <bool>false</bool>
         </property>
         <property name="text" >
          <string>F&amp;orget Fingerprint</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
  <resources/>
 <connections/>
</ui>
